extensions/typescript-language-features/src/features/referencesCodeLens.ts

```diff
@@ -62,7 +62,6 @@ class TypeScriptReferencesCodeLensProvider extends TypeScriptBaseCodeLensProvide
 			case PConst.Kind.const:
```

Comment:
We currently don't any tests for the code lenses on the VS Code since so it'd be a bit of work to add one. If you want to look into this, tests would certainly be nice to have but I don't think they are required here

Reply:
Thank you, I understand. 

I have created the new [issue](https://github.com/microsoft/vscode/issues/85425) (but close it if it shouldn't be there).